### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by pradel**
Add usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,27 @@
...
+yarn add @accounts/mongo
+```
+
+## Usage
+
+```javascript
+import AccountsServer from '@accounts/server';
+import MongoDBInterface from '@accounts/mongo';
+
+# If you are using mongoose
```
Comment (davidyaha) [inline README.md]: Comment mixup :) use `//` instead of `#`

File: README.md [hunk 2]
```diff
@@ -7,3 +7,27 @@
...
+mongoose.connect(process.env.MONGO_URL);
+const db = mongoose.connection;
+
+# If you are using mongodb
```
Comment (davidyaha) [inline README.md]: Here as well

File: README.md [hunk 3]
```diff
@@ -7,3 +7,27 @@
...
+const db =  await mongodb.MongoClient.connect(process.env.MONGO_URL);
+
+AccountsServer.config({
+}, new MongoDBInterface(db));
+```
+
+The users will be saved under the `users` table.
```
Comment (davidyaha) [inline README.md]: collection rather than table

**Comment by pradel** [comment]
Done

**Pull request merged**